### PR TITLE
Minor Oxy pass out code improvement 

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Minor Oxy pass out code improvement, turns redundant code into a proc. I think this is better maybe.

## Why It's Good For The Game
Wow! Its a whole 3 lines shorter!

### CODE REVIEW

**Comment by timothymtorres** [comment]
Small stuff like this improves code readability for programmers and code divers!  Good work!!!

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -631,9 +625,12 @@
 
 /mob/living/carbon/setOxyLoss(amount, updating_health = TRUE, forced = FALSE)
 	. = ..()
-	if(isnull(.))
+	check_passout(.)
+
+/mob/living/carbon/proc/check_passout(oxyloss)
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/carbon_defense.dm:630]: Might as well document this proc

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -631,9 +625,12 @@
...
+	if(isnull(oxyloss))
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/carbon_defense.dm:631]:
```suggestion
	if(!isnum(oxyloss))
```

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by itseasytosee** [comment]
Those three lines I removed return. Give and take.

**Comment by TheBonded** [comment]
> Small stuff like this improves code readability for programmers and code divers! Good work!!!

@timothymtorres dont forget that oranges gave issue jannies permission to label prs as we see fit, remember to use that power for good and to reduce the number of unlabeled prs we have

**Pull request merged**